.github/pull_request_template.md

```diff
@@ -1,13 +1,23 @@
-[ATTENTION. This is the *Pull Request Template*. Replace the two following lines with the indicated information and delete this one. Do not delete anything else]
-
 Closes #[issue number]
 [Description of the changes proposed in the pull request. Include steps to replicate the behavior and screenshots if UI is updated]
 
 # Review checklist
--   [ ] Terms and conditions reflect the current change
--   [ ] Contains enough appropriate tests
--   [ ] If aimed at production, writes a new summary in `whatsnew/whatsnew-pt-PT`
--   [ ] Properly adds an entry in `changelog.md` with the change
--   [ ] If PR includes UI updates/additions, its description has screenshots
--   [ ] Behavior is as expected
--   [ ] Clean, well-structured code
+
+## View Changes
+
+- [ ] Tested both in light and dark mode
+- [ ] Description has screenshots of the UI changes
+- [ ] Text is both in portuguese and english.
+
+## Documentation
+
+- [ ] Properly adds an entry in `changelog.md` with the change
+- [ ] Terms and conditions reflect the current change
+
+## Code
+
+- [ ] No uncessary helper functions
+
+# Release checklist
+
+- [ ] Writes a summary in `whatsnew/whatsnew-pt-PT`
```

Comment:
This is currently not used, I'm not sure we should keep it...

Reply:
It is used in Google Play Store to show users what's new. But as you can realize, this only need to change on releases. I search if there is a possibility to change the PR template based on the branch we are requesting the merge, but I couldn't find anything.

Reply:
I see... Take a look at the other comment, where I suggested changesets and let me now what you think!

Reply:
check the new commit for the release PR